src/EnergyPlus/DataAirflowNetwork.hh

```diff
@@ -1092,6 +1100,57 @@ namespace DataAirflowNetwork {
 
 	};
 
+	struct PressureControllerProp
+	{
+		// Members
+		std::string Name; // Provide a unique object name
+		std::string ZoneName; // Name of the zone that is being controlled
+		int ZoneNum; // Zone number
+		int AFNNodeNum; // AFN node number
+		std::string ControlObjectType; // The control type to be used for pressure control
+		std::string ControlObjectName; // Corresponding control type name
+		int ControlTypeSet; // Control type set to be used for pressure control
+		int AvailSchedPtr; // Availability schedule pointer
+		int PresSetpointSchedPtr; // Pressure setpoint schedule pointer
+
+		// Default Constructor
+		PressureControllerProp( ) :
+			ZoneNum( 0 ),
+			AFNNodeNum( 0 ),
+			ControlTypeSet( 0 ),
+			AvailSchedPtr( 0 ),
+			PresSetpointSchedPtr( 0 )
+		{}
+
+	};
+
+	struct DisSysCompAirflowProp // OA fan component
+	{
+		// Members
+		std::string Name; // Name of exhaust fan component
```

Comment:
@mjwitte No. This struct is used for both objects of AirflowNetwork:Distribution:Component:OutdoorAirFlow and AirflowNetwork:Distribution:Component:ReliefAirFlow.